proto/topo.proto

```diff
@@ -81,6 +81,34 @@ message Node {
   // If interfaces is empty the interfaces defined in the links portion of the
   // topology will be populated into the node.
   map<string, Interface> interfaces = 12;
+  // Any host vm constraints required by the vendor.
+  repeated HostConstraint host_constraints = 13;
+}
+
+// HostConstraint includes all the necessary system constraints required 
+// which could be a sysctl setting/cgroup
+message HostConstraint{
+  oneof constraints{
```

Comment:
I think this is better as singular

HostConstraint.GetConstraint() since each of these messages contains a single constraint only

Reply:
Thanks for the suggestion, fixed.